server-spi-private/src/main/java/org/keycloak/requestfilter/IpAccessFilterProvider.java

```diff
@@ -0,0 +1,8 @@
+package org.keycloak.requestfilter;
+
+public class IpAccessFilterProvider implements RequestFilterProvider {
```

Comment:
Do you need this empty class or can you just return `null` in `IpAccessFilterProviderFactory#create()`?

Alternatively you could create a singleton instance of this provider in `IpAccessFilterProviderFactory`  and return that in `IpAccessFilterProviderFactory#create()`.